contracts/src/base/registry/facets/distribution/v2/StakingRewards.sol

```diff
@@ -0,0 +1,329 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+// interfaces
+
+// libraries
+import {FixedPointMathLib} from "solady/src/utils/FixedPointMathLib.sol";
+import {SafeTransferLib} from "solady/src/utils/SafeTransferLib.sol";
+import {CustomRevert} from "contracts/src/utils/libraries/CustomRevert.sol";
+
+// contracts
+import {DelegationMinion} from "./DelegationMinion.sol";
+
+library StakingRewards {
+  using CustomRevert for bytes4;
```

Comment:
I'd rather use the library explicitly, open to comments as to why this is more useful

Reply:
Syntax sugar.